raiden/tests/integration/api/test_restapi.py

```diff
@@ -1317,11 +1325,35 @@ def test_payment_events_endpoints(api_backend, raiden_network, token_addresses):
     assert len(response) == 2
     assert response[0]['event'] == 'EventPaymentSentSuccess'
     assert response[1]['event'] == 'EventPaymentSentSuccess'
+    # test endpoint without (partner and token) for target1
+    request = grequests.get(
+        api_url_for(
+            app1_server,
+            'paymentresource',
+        ),
+    )
+    response = request.send().response
+    assert_proper_response(response, HTTPStatus.OK)
+    response = response.json()
+    assert len(response) == 1
```

Comment:
I don't think we should rely on the quantity/order of events, what about using `must_have_event`?

Reply:
Sure, I can do that.